### ISSUE TITLE
How to modify the output font of markdown cells

### ISSUE THREAD

**Issue by daiannan1993** [wallneradam/jupyterlab-custom-css#1]
Hello and so lucky to meet this repo so that I could get the output font of code cells modified to the specific style I want. 

And as not familiar with css, I'm being concerned about how to change the font of markdown cells output so that all the outputs could have consistent outlook. I looked it up in the web and was told that the term might be "jp-MarkdownOutput" or "jp-RenderedMarkdown", which didn't work out right. 

Hopefully I could get some hints from here. THX.

**Comment by daiannan1993** [comment]
the two terms both work fine...

**Comment by wallneradam** [comment]
Hi, you may not need this extension for changing font of HTML outputs, it is built in.
You can use Settings/Theme/overrides for this. This in the background set CSS variables, and HTML font family is a CSS variable.
So the line `"jp-content-font-family": "YourPreferredFontName"` should do the trick.
Though if you want to set fonts just for markdown outputs, my extension with the CSS classes you found are the solution.

**Comment by daiannan1993** [comment]
Yeah actually I made a mistake when I tried to set the font-family term which I found out later that I wrote as "font-size". 
And your extension really helps as I had tried to modify some variables in the config files of jupyter in the system or in different envs. Lots of time and efforts had been taken before I encountered your solution. Thankfully it helps ease my OCD hhhh
Thanks again for the reply ><
